modal-js/test/sandbox_filesystem.test.ts

```diff
@@ -0,0 +1,201 @@
+import { App } from "modal";
+import { expect, test, onTestFinished } from "vitest";
+
+test("WriteAndReadTextFile", async () => {
```

Comment:
See comment in Go SDK about trying to keep the tests identical between TS / Go just so we equally good identical test coverage in each.